helm/coredns-app/templates/configmap.yaml

```diff
@@ -46,6 +42,38 @@ data:
         {{- end }}
     }
 
+    {{- $domains := split " " .Values.cluster.kubernetes.clusterDomain }}
```

Comment:
I'd really like to see a list here instead, but it might complicate the upgrade. not sure if it's worth to do some helm magic to make it work. otherwise looks good.

Reply:
You're right, let me create a follow-up issue. Currently I don't know how to do that without breaking upgrades